tools.go

```diff
@@ -4,5 +4,5 @@ package tools
 
 import (
 	_ "github.com/golang/mock/mockgen@v1.4.4"
-	_ "sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.0"
+	_ "sigs.k8s.io/controller-tools/cmd/controller-gen@v0.8.0"
```

Comment:
We can remove the entire tools.go file. `controller-gen` comes through `make controller-gen` in kubebuilder 3 and mockgen we can add via 
```
MOCKGEN = $(shell pwd)/bin/mockgen
.PHONY: mockgen
mockgen: ## Download mockgen locally if necessary.
	$(call go-get-tool,$(MOCKGEN),github.com/golang/mock/mockgen@v1.6.0)
```
to the make file